packages/list-exports/package.json

```diff
@@ -84,6 +84,6 @@
 		"safe-publish-latest": "^2.0.0"
 	},
 	"engines": {
-		"node": "^18.17.0 || >=20.5.0"
+		"node": ">= 10"
```

Comment:
oh, i may already have a breaking change queued up, in which case this should be left alone :-)